{
    "patch_is_None": [
        "sympy__sympy-20801"
    ],
    "patch_exists": [
        "django__django-11603",
        "django__django-12143",
        "django__django-13590",
        "django__django-13741",
        "django__django-13810",
        "django__django-14089",
        "django__django-14373",
        "django__django-14500",
        "django__django-15467",
        "django__django-16493",
        "django__django-16612",
        "scikit-learn__scikit-learn-10297",
        "scikit-learn__scikit-learn-11578"
    ],
    "patch_successfully_applied": [
        "django__django-11603",
        "django__django-12143",
        "django__django-13590",
        "django__django-13741",
        "django__django-13810",
        "django__django-14089",
        "django__django-14373",
        "django__django-14500",
        "django__django-15467",
        "django__django-16493",
        "django__django-16612",
        "scikit-learn__scikit-learn-10297",
        "scikit-learn__scikit-learn-11578"
    ],
    "resolved": [
        "django__django-11603",
        "django__django-12143",
        "django__django-13590",
        "django__django-13741",
        "django__django-14089",
        "django__django-14373",
        "django__django-14500",
        "django__django-15467",
        "scikit-learn__scikit-learn-11578"
    ],
    "unresolved": [
        "django__django-13810",
        "django__django-16493",
        "django__django-16612",
        "scikit-learn__scikit-learn-10297"
    ],
    "image_build_error": [
        "pydata__xarray-4966",
        "sympy__sympy-20801"
    ]
}